tests/libs/util/native_helper.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include "catch_wrapper.hpp"
+#include "ebpf_execution_type.h"
+#include "native_helper.hpp"
+
+#include <rpc.h>
+
+static std::string
+_guid_to_string(_In_ const GUID* guid)
+{
+    char guid_string[37] = {0};
+    sprintf_s(
```

Comment:
Can't you just use https://learn.microsoft.com/en-us/windows/win32/api/rpcdce/nf-rpcdce-uuidtostringa instead of reimplementing it?   It's exposed by the same header/library as UuidCreate() which you already call.

Reply:
Fixed.